packages/core/src/models/uischema.ts

```diff
@@ -249,3 +258,15 @@ export const isGroup = (layout: Layout): layout is GroupLayout =>
 
 export const isLayout = (uischema: UISchemaElement): uischema is Layout =>
   (uischema as Layout).elements !== undefined;
+
+export const isScopeable = (obj: object): obj is Scopable =>
+  obj !== undefined && obj.hasOwnProperty('scope');
+
+export const isScoped = (obj: object): obj is Scoped =>
+  isScopeable(obj) && obj.scope !== null;
+
+export const isLabelable = (obj: object): obj is Lableable =>
+  obj !== undefined && obj.hasOwnProperty('label');
+
+export const isLabeled = (obj: object): obj is Labeled =>
+  isLabelable(obj) && obj.label !== null;
```

Comment:
Here the inconsistent becomes apparent. Why allow `null` values for `scope` but not for `label`.

Reply:
It should be checked against undefined here too.